src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -622,42 +622,33 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 		autotune_attitude_control_status_s status{};
```

Comment:
I think we should check command validity here - i.e. if we're touching this anyway, pull the changes I put in around https://github.com/PX4/PX4-Autopilot/pull/25795/files#r2449948304 that check the result and cause the ACK ( but with your much better value checking)

See also https://github.com/PX4/PX4-Autopilot/pull/25797#discussion_r2476111958